### ISSUE TITLE
Setting mask value to 1 when no face detected 

### ISSUE THREAD

**Issue by ArmastusChen** [Francis-Rings/StableAnimator#44]
Thanks for your great work.

I am looking at the preprocessing code for face mask generation. When the face is not detected, all mask value is set to 1 (please refer to links below). Is this intended or just a typo? It seems setting them to 0 makes more sense to me. Thank you in advance. 

https://github.com/Francis-Rings/StableAnimator/blob/65a72f8702cb08160bcd1a23b4bbd8a6042487aa/face_mask_extraction.py#L36

https://github.com/Francis-Rings/StableAnimator/blob/65a72f8702cb08160bcd1a23b4bbd8a6042487aa/animation/dataset/animation_dataset.py#L184C17-L184C30

**Comment by Francis-Rings** [comment]
Hi, the face masks are primarily used to calculate the reconstruction loss in the face-related regions during the training phase. When the face detector fails to detect the human face, I intend to set the values of the synthesized masks to 1 to calculate the overall video reconstruction loss.

**Comment by ArmastusChen** [comment]
Thanks for you quick response. I am also wondering the image color of detected dwpose is different than that the one detected by original repo.  Seems like the dwpose detected in this repo is saved in BGR format. I should follow this BGR format when fine-tuning the model right?

**Comment by ArmastusChen** [comment]
Actually, I inputted both the dwpose images with RGB and BGR format into the pretrained model. The results are pretty similar. Could you please confirm which did you use for training? Thanks

**Comment by Francis-Rings** [comment]
Hi, please utilize `DWPose/training_skeleton_extraction.py` to extract human skeleton poses for training your StableAnimator. For more details, please refer to the Model Training section in the README file. By the way, StableAnimator-Pro is still under review, and I will release its weights as soon as the review process is complete.
